src/store/rescan.js

```diff
@@ -42,13 +94,24 @@ export default {
         return false;
       }
     },
-    async start({ commit, dispatch, rootState }) {
+    async startAll({ commit, dispatch, rootState }) {
+      commit("setLastClick", new Date());
+      try {
+        await api.rescans.startAll(rootState.auth);
+        setTimeout(() => dispatch("index"), 2500);
+        return true;
+      } catch (error) {
+        commit("addError", error, { root: true });
+        return false;
+      }
+    },
+    async start({ commit, dispatch, rootState }, id) {
       commit("setLastClick", new Date());
       try {
-        const result = await api.rescan.start(rootState.auth);
+        const result = await api.rescans.start(rootState.auth, id);
         result.running = true;
         commit("setRescan", result);
```

Comment:
This fails here for me. `setRescan` expects something of the shape `{ id, rescan }`, but you just pass `result` here. (Which has an `id` property, but not a `rescan` property.)